widget/embedded/src/components/SwapsGroup/SwapsGroup.tsx

```diff
@@ -57,68 +102,68 @@ export function SwapsGroup(props: PropTypes) {
   }
 
   return (
-    <>
-      {groups
-        .filter((group) => group.swaps.length > 0)
-        .map((group) => (
-          <React.Fragment key={group.title}>
-            <Group>
-              <Time>
-                <Typography
-                  variant="label"
-                  size="medium"
-                  className={groupStyles()}>
-                  {i18n.t(group.title)}
-                </Typography>
-              </Time>
-              <Divider size={4} />
-              <SwapList>
-                {group.swaps.map((swap) => {
-                  const firstStep = swap.steps[0];
-                  const lastStep = swap.steps[swap.steps.length - 1];
-                  return (
-                    <React.Fragment key={swap.requestId}>
-                      <SwapListItem
-                        requestId={swap.requestId}
-                        creationTime={swap.creationTime}
-                        status={swap.status}
-                        onClick={onSwapClick}
-                        onlyShowTime={group.title === 'Today'}
-                        swapTokenData={{
-                          from: {
-                            token: {
-                              image: firstStep.fromLogo,
-                              displayName: firstStep.fromSymbol,
-                            },
-                            blockchain: {
-                              image: firstStep.fromBlockchainLogo || '',
-                            },
-                            amount: limitDecimalPlaces(swap.inputAmount),
-                          },
-                          to: {
-                            token: {
-                              image: lastStep.toLogo,
-                              displayName: lastStep.toSymbol,
-                            },
-                            blockchain: {
-                              image: lastStep.toBlockchainLogo || '',
-                            },
-                            amount: limitDecimalPlaces(
-                              lastStep.outputAmount || ''
-                            ),
-                            estimatedAmount: limitDecimalPlaces(
-                              lastStep.expectedOutputAmountHumanReadable || ''
-                            ),
-                          },
-                        }}
-                      />
-                    </React.Fragment>
-                  );
-                })}
-              </SwapList>
-            </Group>
-          </React.Fragment>
-        ))}
-    </>
+    <GroupedVirtuoso
+      endReached={() => {
+        if (loadedItems.current < list.length) {
+          loadMore();
+        }
+      }}
+      groupCounts={currentGroupCounts}
+      groupContent={(index) => {
+        return (
+          <Group hasMargin={index !== 0}>
```

Comment:
Can we use a divider instead of setting margin?

Reply:
The divider component does not yield the desired outcome.